WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -54,7 +57,7 @@ public BitcoinTabSettingsViewModel()
 				.Subscribe(_ => Save());
 		}
 
-		public Version BitcoinCoreVersion => Constants.BitcoinCoreVersion;
+		public Task<string> BitcoinKnotsVersion => GetNodeVersionAsync();
```

Comment:
How about renaming the variable to:

```suggestion
		public Task<string> BitcoinFullNodeVersion => GetNodeVersionAsync();
```

Reply:
This variable name won't be visible to users who do not read code. It was Core earlier even though we support Knots. Core is still used at lot of places so its okay to use Knots here.

Reply:
This is not about the built-in node binary that we ship, if that was the case then there is no reason for this PR in the first place and hard coding the version is enough.
This represents the full node version that the user is using, it might be Core or Knots or anything else.

> It was Core earlier even though we support Knots. Core is still used at lot of places so its okay to use Knots here.

If naming the variable with Core is wrong then it should be fixed, not using Core with some variables and Knots with other ones. Consistency.

Reply:
There is no consistency in the name of variables used for full node.

Reply:
No need to add more inconsistency then.

Reply:
I prefer Bitcoin Knots in this variable name. You are free to NACK the PR or open a separate PR.

Reply:
Done